### ISSUE TITLE
Make number line draggable handle more discoverable

### ISSUE THREAD

**Issue by catherinecarter** [phetsims/number-pairs#43]
During interviews, all three students struggled to find the interactive component on the number line (i.e. the scrubber). The following changes are proposed: 

1. Remove the color behind each addend and the total (with color, they look interactive). Replace with a transparent white color in case the number is in front of other objects.
2. Have the color appear only when the handle is moved, like in the accordion box in [Number Line: Integers](https://phet.colorado.edu/sims/html/number-line-integers/latest/number-line-integers_all.html?screens=2)
3. Make the scrubber handle larger
4. Make the pink color on the scrubber handle darker

**Comment by catherinecarter** [comment]
Do you mean when the user adds or subtracts one from either addend? If so, I think it would make the action more clear if the addend being changed had a color appear to show what was happening. 

What do you mean changing from one scene to another (on the Sum screen)?

**Comment by marlitas** [comment]
@catherinecarter changing from one scene to another on the other screens. Sorry that was unclear. 

Also...

> Make the pink color on the scrubber handle darker

What color would you like to use instead? You should be able to adjust it in the color editor if you want to look at it in real time.

**Comment by marlitas** [comment]
Realized I need an answer on wether we want the colors to appear during scene changes in the first three screens before I can move forward on this. @catherinecarter let me know what you would like me to do.

**Comment by catherinecarter** [comment]
No, I don't think so. During interviews, students wanted to touch and interact with the addends and total numbers, so removing the colored background is an attempt to make those less interactive. 

I would be concerned that by adding color upon changing scenes, a user would be drawn to those numbers again and think they're interactive.

**Comment by catherinecarter** [comment]
Decisions after design meeting 02/13/25
* Make the rectangle have smaller dimensions so the stick connecting the number line dot and the draggable handle is still visible
* Make the opacity on the rectangle behind the number 1.0 (opaque)
* Do not have a dynamically appearing highlight behind the addends when the handle is moved

**Comment by marlitas** [comment]
Decisions applied above. @catherinecarter can you review and close if all looks well?

**Comment by catherinecarter** [comment]
I really like how we can't see the number behind it anymore, but I do miss seeing the stick part connecting the number line to the scrubber. Can we add another rectangle behind the tick number but in front of the opaque rectangle and add a 50% opacity stick part behind it?

**Comment by marlitas** [comment]
> Can we add another rectangle behind the tick number but in front of the opaque rectangle and add a 50% opacity stick part behind it?

I'm not sure what you're trying to describe here. I'm quite confused.

**Comment by catherinecarter** [comment]
Just when you think something is so clear in your head haha.

What I mean is this (hopefully this is more clear and I'll include what I *think* is happening already):
* Right now this is the order from back to front (yes?)
   a. The tick mark label is in the back
   b. There's an opaque rectangle in front of the tick number to hide it
   c. The number associated with the `thumbNode` is on top of the opaque rectangle

* I'm proposing to add a second rectangle between (b) and (c) that has the stick part in the center to match the stick that's being hidden by (b). When taken apart, (a), (b), and (c) are shown in order from left to right.
* Like this: 
   * <img width="200" alt="Image" src="https://github.com/user-attachments/assets/1e9cc6e4-b5da-460f-a367-38d45aeb5716" />
* So when you put all the pieces together, it looks like the stick can be seen but the number line tick mark is hidden. Something like this:
   * <img width="200" alt="Image" src="https://github.com/user-attachments/assets/42789279-93b3-4706-85fd-0d744a17578f" />

My goal is to still hide the tick mark number while showing a reduced-opacity line from the number line to the `thumbNode`.

Maybe you know an easier way to do that? And hopefully this builds clarity?

**Comment by marlitas** [comment]
Hmmm I think that this has the potential to really not line up properly and be a headache down the line... My instinct is to push back on this, and if that is really necessary we should perhaps look at a different solution in general.

**Comment by catherinecarter** [comment]
I remember @kathy-phet suggested this as a solution and wonder what wouldn't line up properly?

**Comment by kathy-phet** [comment]
@catherinecarter - I thought the solution where we just had the quite opaque, but tight, white background behind the number was working pretty well? I had thought about a white layer but not a white layer with a line in it - that I recall.

**Comment by catherinecarter** [comment]
Ok, @kathy-phet, maybe I didn't remember quite correctly. I will send to a few people and see if the scrubber is more discoverable now. 

I'll consider this issue on hold for now (not more than a couple days). Thanks!

**Comment by marlitas** [comment]
@catherinecarter and I talked through this together, and arrived at a design that we both feel good about. The label is now above the number line matching what you see in Number Line Operations.

<img width="152" alt="Image" src="https://github.com/user-attachments/assets/2d6b7781-3d56-4874-a400-0bc016e2a71c" />

I believe this issue is now ready to close.
